<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QDialog" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 white, stop: 1 #eaeaea);
 border: 1px solid #cecece;
 border-radius: 10px;
font-size: 12px;
}
QGroupBox {
 border: 1px solid #cecece;
border-radius: 10px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #4193ff, stop: 1 #9eeaff);
}
 QPushButton {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 white, stop: 1 #eaeaea);

 border: 0.5px solid rgb(206, 206, 206);
 }
QLabel {
	background-color: rgb(193, 193, 193);
}</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>611</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <column>
    <property name="text">
     <string>Inventory number</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Model</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Year production</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Life time</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_OUTPUT">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>63</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Output:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_OUTPUT">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(193, 193, 193);
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_View">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>View all data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SY">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Since year...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SM">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Count model</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GRAPHIC">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Production Graphic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SD">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>D/d trams</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_INPUT">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_BACK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 background-color: rgb(193, 193, 193);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #4193ff, stop: 1 #9eeaff);
}</string>
   </property>
   <property name="text">
    <string>Back to Main</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>51</y>
     <width>601</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
